ksqldb-execution/src/main/java/io/confluent/ksql/execution/interpreter/TermCompiler.java

```diff
@@ -191,7 +192,7 @@ public Term visitInPredicate(
       final Context context
   ) {
     final InPredicate preprocessed = InListEvaluator
-        .preprocess(inPredicate, expressionTypeManager);
+        .preprocess(inPredicate, expressionTypeManager, Collections.emptyMap());
```

Comment:
Why don't you get the lambda mapping from the context here?

Reply:
cc @AlanConfluent for a quick look.

Reply:
that's a miss, I'll put in the lambda mapping